core/__init__.py

```diff
@@ -29,6 +29,13 @@ def sv_unregister_modules(modules):
     for m in reversed(modules):
         if hasattr(m, "unregister"):
             # print("Unregistering module: {}".format(m.__name__))
+            if m.__name__ == "sverchok.old_nodes":
```

Comment:
Are you sure that old modules visit this method? I guess that there register unregister methods are only called in `load_post` handler.

Reply:
i am , because otherwise `m.__name__ == "sverchok.old_nodes"`  would not be visisted, but it is.

but i think the solution to fix the `unregister` function inside old_nodes, so it checks if a node.is_registered_node_type or is `hasattr(bpy.types, node.bl_idname)`

it should be able to call unregister, without raising an exception -- even if all nodes are unregistered. -- maybe it can print a simple message, that only few of us would ever see, but we can search the code base if we see it.

Reply:
Can you say how the problem can be reproduced? When I use reload (F8) I can't see any prints about errors that a module can't be unregistered.

Reply:
look at the output of 'test it` https://github.com/nortikin/sverchok/runs/2764532114   at the very end